schema/me/conversation/index.js

```diff
@@ -272,14 +272,25 @@ export const ConversationFields = {
   messages: {
     type: MessageConnection,
     description: "A connection for all messages in a single conversation",
-    args: pageable(),
+    args: pageable({
+      sort: {
+        type: new GraphQLEnumType({
+          name: "sort",
+          values: {
+            DESC: { value: "desc" },
+            ASC: { value: "asc" },
+          },
+        }),
+      },
+    }),
     resolve: ({ id, from_email }, options, req, { rootValue: { conversationMessagesLoader } }) => {
       const { page, size, offset } = parseRelayOptions(options)
       return conversationMessagesLoader({
         page,
         size,
         conversation_id: id,
         "expand[]": "deliveries",
+        sort: options.sort || "asc",
```

Comment:
Since this is enforced at the server level, we probably can just leave this out altogether. Alternatively, if we did want to add this, we could do it via `deprecationReason` on the `sort` arg definition.

Reply:
I thought about it as well and it's just less code including a default value. If we don't and `options.sort` is `null` or `undefined` it's gonna set it to some garbage like `&sort=undefined`

Reply:
Oh yea good call, that sounds annoying.